listen_test.go

```diff
@@ -145,6 +147,46 @@ func Test_Listen_TLS(t *testing.T) {
 	}))
 }
 
+// go test -run Test_Listen_Acme_TLS
+func Test_Listen_Acme_TLS(t *testing.T) {
+	dir, err := os.Getwd()
+	require.NoError(t, err)
+	dir, err = os.MkdirTemp(dir, "certs")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir) //nolint:errcheck // ignore error
+
+	// Certificate manager
+	m := &autocert.Manager{
+		Prompt: autocert.AcceptTOS,
+		// Replace with your domain
+		HostPolicy: autocert.HostWhitelist("example.com"),
+		// Folder to store the certificates
+		Cache: autocert.DirCache(dir),
+		// Define the Client for test
+		Client: &acme.Client{
+			DirectoryURL: "https://acme-staging-v02.api.letsencrypt.org/directory",
+		},
+	}
+
+	app := New()
+
+	// invalid port
+	require.Error(t, app.Listen(":99999", ListenConfig{
```

Comment:
@ReneWerner87 @efectn This is not failing, which may have exposed a bug in `Fiber`. That's an invalid port.

Reply:
@gaby can we deal with this in an extra bug and change the port for this

Reply:
Error was confirmed, reported here https://github.com/gofiber/fiber/issues/3237